core/src/main/cljc/jarl/config/reader.cljc

```diff
@@ -0,0 +1,24 @@
+(ns jarl.config.reader
+  (:require [jarl.encoding.json :as json]
+            [jarl.encoding.yaml :as yaml]
+            [jarl.runtime.environment :as environment]
+            [camel-snake-kebab.core :as csk]
+            [clojure.string :as str]))
+
+(defn- env->resolved [value]
+  (let [env-var (-> (first value) (str/replace #"\$\{" "") (str/replace #"\}" ""))]
+    (or (environment/env-var env-var) "")))
```

Comment:
I wonder, would it be a good idea to leave the var name in place if the var can't be found in the env? Might help troubleshooting. It'd be a bit more of a hassle if you wanted the empty-string default behavior; you'd basically need to assign an empty string to the env var (or we'd need a flag to enable that behavior).

Reply:
Yeah, that might be helpful. At this point I was just going for OPA compatibility.

![Screenshot 2022-10-27 at 14 04 51](https://user-images.githubusercontent.com/510711/198280059-bd496a68-387a-437a-9992-98a1f295f9fb.png)